test/observer.spec.ts

```diff
@@ -691,10 +691,6 @@ describe('DataLayerObserver unit tests', () => {
     expect(observer.handlers.length).to.eq(1);
     expect(globalMock.dataLayer.length).to.eq(0);
 
-    // Ignore the first two as they are observation messages
-    expectParams(appender, 'log');
-    expectParams(appender, 'log');
```

Comment:
These 2 ignored calls were for `record` level log entries which have been removed.